public/app/features/dashboard/components/AddPanelWidget/AddPanelWidget.tsx

```diff
@@ -297,22 +297,22 @@ const getStyles = (theme: GrafanaTheme) => {
       height: 38px;
       flex-shrink: 0;
       width: 100%;
-      font-size: ${theme.typography.size.md};
-      font-weight: ${theme.typography.weight.semibold};
-      padding-left: ${theme.spacing.sm};
+      font-size: ${theme.v1.typography.size.md};
```

Comment:
I think this could be `theme.typography.body.fontSize`

Reply:
That's `1rem` right which is `16px`. It used to be `14px` so i think `theme.typography.fontSize` would be better.

Reply:
Or maybe `theme.typography.bodySmall.fontSize`. That should roughly be `14px` right?